namer/configuration_utils.py

```diff
@@ -141,7 +141,9 @@ def from_config(config: ConfigParser) -> NamerConfig:
     namer_config.enable_metadataapi_genres = config.getboolean("metadata", "enable_metadataapi_genres", fallback=False)
     namer_config.default_genre = config.get("metadata", "default_genre", fallback="Adult")
     namer_config.language = config.get("metadata", "language", fallback=None)
-    namer_config.ignored_dir_regex = config.get("metadata", "ignored_dir_regex", fallback=".*_UNPACK_.*")
+    ignored_dir_regex = config.get("metadata", "ignored_dir_regex", fallback=r".*_UNPACK_.*")
```

Comment:
I think the problem is it's got the metadata section, not namer section.

Reply:
lol, it's `watchdog` actually, https://github.com/ThePornDatabase/namer/blob/main/namer.cfg.sample#L170